iOS Dev Calendar/Calendar/Event/EventsView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  EventsView.swift
+//  iOS Dev Calendar
+//
+//  Created by Wesley Keetch on 4/22/25.
+//
+
+
+import SwiftUI
+
+struct EventsView: View {
+  let date: Date
+  let events: [Date: [Event]]
+  
+  var body: some View {
+    VStack(alignment: .leading, spacing: 16) {
+      Text(title)
+        .font(.system(size: 20, weight: .bold))
```

Comment:
You should start using a view modifier for font so you can swap out the font for the entire app in one place.